poppy/poppy_core.py

```diff
@@ -2273,8 +2273,10 @@ def get_phasor(self,wave):
                 lx,ly=resampled_amplitude.shape
                 #crop down to match size of wavefront:
                 lx_w,ly_w = wave.amplitude.shape
-                border_x = np.abs(np.floor((lx-lx_w)/2))
-                border_y = np.abs(np.floor((ly-ly_w)/2))
+
+                # explicit cast to int fixes index errors in Numpy >= 1.12.0
+                border_x = int(np.abs(np.floor((lx-lx_w)/2)))
```

Comment:
Ah, but on the other hand: 
```
In [1]: lx_w, lx = 45, 20

In [2]: int(np.abs(np.floor((lx-lx_w)/2)))
Out[2]: 13

In [3]: (lx - lx_w) // 2
Out[3]: -13
```

We know the shape elements are integers but we don't know which is larger.  So I think the absolute value is necessary.